go/generate.go

```diff
@@ -8,4 +8,11 @@ import (
 )
 
 //go:generate go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
-//go:generate protoc --proto_path=../proto -I=. -I=../build/tmp/vendor --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --grpc-gateway_out=generate_unbound_methods=true,paths=source_relative:. ../proto/message_api/v1/message_api.proto ../proto/message_api/v1/authn.proto ../proto/message_contents/xmtp_envelope.proto
+//go:generate -command compile protoc --proto_path=../proto -I=. -I=../build/tmp/vendor --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --grpc-gateway_out=generate_unbound_methods=true,paths=source_relative:.
```

Comment:
Trying to split up this unmaintainable line (apparently we were already missing private key and composite here). This line defines an "alias" called "compile" that translates to the rest of this line (i.e. all the protoc with all the parameters). Then we can use the alias to compile the proto files individually, making the list of files clearer.

Reply:
We could also have it just run a script that is multi-line

Reply:
In that case it would probably make more sense to skip go:generate altogether. I'll leave as is, we can decide to switch in separate PR, for now I just wanted to make this more maintainable.